src/Integration.Vsix/CFamily/PreCompiledHeaders/PreCompiledHeadersEventListener.cs

```diff
@@ -100,6 +104,16 @@ public void Dispose()
             {
                 activeDocumentTracker.OnDocumentFocused -= OnActiveDocumentFocused;
                 activeDocumentTracker?.Dispose();
+
+                try
```

Comment:
Dispose() does gets called when VS shuts down so it's a good place to delete the files.